.buildkite/commands/run-tests.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -eu
+
+SIMULATOR_NAME=$1
+
+xcodebuild clean test \
+  -project 'XCUITestHelpers.xcodeproj' \
+  -scheme 'XCUITestHelpers' \
+  -destination "platform=iOS Simulator,name=$SIMULATOR_NAME,OS=14.5"
```

Comment:
See rationale from [this comment](https://github.com/Automattic/ScreenObject/pull/6/files#r695335144) on why this doesn't use Fastlane and has the OS value hardcoded.

Reply:
I wonder if it's worth a `# comment` in the `.sh` file itself 🤔